### ISSUE TITLE
Reader: Jetpack Tiled Gallery block not displaying properly.

### ISSUE THREAD

**Issue by elibud** [Automattic/wp-calypso#45122]
#### Steps to reproduce

1. Starting at URL: https://wordpress.com/post
2. Create a post with a Jetpack Tiled Gallery
3. Publish the post
4. View the post in the Reader

#### What I expected

To see a properly formatted gallery on Reader.

#### What happened instead

A list of images is displayed.

#### Browser / OS version
Safari 13.1.2

#### Screenshot / Video
https://wordpress.com/read/feeds/67062177/posts/2869581116

**Comment by wynwin** [comment]
Another report here: 3309874-zen

**Comment by kwight** [comment]
I don't see any remaining issues @elibud @yaelirub :

<img width="1125" alt="Screen Shot 2020-10-06 at 3 41 50 PM" src="https://user-images.githubusercontent.com/349751/95267633-8e88be00-07ea-11eb-9d5e-413da27a4a67.png">

**Comment by elibud** [comment]
Your post looks great @kwight but the one I linked to in the original report still looks like a list of images, maybe some gallery setting?

**Comment by kwight** [comment]
Ah! Different gallery: yours is a Jetpack Tiled Gallery, I was testing the core gallery. Updated the issue title.

**Comment by elibud** [comment]
My bad, thank you!

**Comment by nerdysandy** [comment]
Just to add (in case this detail gets missed when replicating the issue) the issue is when you view the post by clicking on it from the Reader.

**Comment by davipontesblog** [comment]
Part of the Block Audit milestone: https://github.com/Automattic/wp-calypso/milestone/320
As discussed with @4thhubbard, leaving it for future consideration.

**Comment by yansern** [comment]
Just to add some visual context:

![2021-05-19_11-27-56](https://user-images.githubusercontent.com/1287077/118789737-5d4e9100-b895-11eb-8b1f-e79af289f53d.jpg)
![2021-05-19_11-27-42](https://user-images.githubusercontent.com/1287077/118789742-5f185480-b895-11eb-9030-5f1cd5e3aef6.jpg)

**Comment by yansern** [comment]
Technical analysis: Jetpack carousel CSS is not loaded on Reader view. The question is whether this was intentional.

Eitherway, the two CSS that should be loaded are:
- wp-content/mu-plugins/carousel/jetpack-carousel.css
- wp-content/mu-plugins/jetpack/_inc/blocks/tiled-gallery/view.css

**Comment by jerrysarcastic** [comment]
Also reported in https://wordpress.com/forums/topic/reader-shows-pictures-without-space-between

**Comment by karenroldan** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – Replicated 

📌 FINDINGS/SCREENSHOTS/VIDEO
- The issue can be replicated exactly as described above. 

**View on the live page**
![5jSmoz.png](https://github.com/Automattic/wp-calypso/assets/67279475/eec7682a-9ca2-49a8-8985-c5293c7ac248)

**View from the Reader**
![Lh62XB.png](https://github.com/Automattic/wp-calypso/assets/67279475/8f139c67-f33a-4c3c-ab71-475deb620c36)

**Comment by sixhours** [comment]
Given the number of available options and possible configurations in the Tiled Gallery block, I'm not sure it makes sense to try to recreate the gallery exactly for Reader's purposes. The Tiled Gallery uses a lot of external JS that we'd need to duplicate in Calypso (and keep maintained) to make that happen.

Unless there's a publicly accessible Tiled Gallery library/API we can use somewhere? I noticed we did something similar for the old version of Jetpack Slideshows here: 

fbhepr%2Skers%2Spnylcfb%2Spyvrag%2Spbzcbaragf%2Srzorq%2Qpbagnvare%2Svaqrk.wfk%3Se%3Q58nn7nr7%2323%2Q28-og

I'll look into a Tiled Gallery library. But I think finding an acceptable middle ground is the key -- make the images look like a gallery, even if it's not a proper tiled gallery. That improves on the existing view.
